### PR TITLE
Duplicate IDs changed

### LINKED ISSUE #933: Duplicate IDs, Aria-labels and Labels in Example and test files.
## Description
Examples and Test Files of certain components have duplicate IDs, Aria-labels and label , thus giving violations while running the axe plugin.

## Components

- Date   Picker
- Date Time Picker
- Modal Manager
- Navigation Prompt
- Application Navigation
- Slide Panel Manager

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Fixes #933

### CODE REVIEW

File: packages/terra-application-navigation/CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ Unreleased
 ### Fixed
 * Fix wdio test url.
 
+### Changed
+* Duplicate ID in example changed.
```
Comment (avinashg1994) [inline packages/terra-application-navigation/CHANGELOG.md]: This should be moved to the unreleased section.

Comment (saket2403) [reply packages/terra-application-navigation/CHANGELOG.md]: Resolved in 8f95e89

File: packages/terra-date-time-picker/src/terra-dev-site/doc/guides/DateTimePickerEvents.jsx [hunk 2]
```diff
@@ -67,22 +67,22 @@ class DateTimePickerEvents extends React.Component {
...
           onClickOutside Trigger Count:
           {' '}
-          <span id="blur-state">{this.state.clickOutsideTriggerCount}</span>
+          <span id="onClickOutside-count">{this.state.clickOutsideTriggerCount}</span>
           <br />
           <br />
           onChangeRaw Triggered:
           {' '}
-          <span id="blur-state">{this.state.changeRawValue}</span>
+          <span id="onChnageRaw-count">{this.state.changeRawValue}</span>
```
Comment (avinashg1994) [inline packages/terra-date-time-picker/src/terra-dev-site/doc/guides/DateTimePickerEvents.jsx]: Typo!

File: packages/terra-date-time-picker/src/terra-dev-site/doc/guides/DateTimePickerEvents.jsx [hunk 3]
```diff
@@ -57,7 +57,7 @@ class DateTimePickerEvents extends React.Component {
         <p>
           onFocus Trigger Count:
           {' '}
-          <span id="blur-state">{this.state.focusTriggerCount}</span>
+          <span id="focus-state">{this.state.focusTriggerCount}</span>
```
Comment (avinashg1994) [inline packages/terra-date-time-picker/src/terra-dev-site/doc/guides/DateTimePickerEvents.jsx]: nit: Just to keep it consistent with the other IDs, you could changed the id to **"onFocus-count"** and also for blur count to **"onBlur-count"**

**Comment by avinashg1994** [comment]
Not sure, but should the IDs for DateTimePicker in the files "DateTimePickerIncomplete.jsx" and "DateTimePickerInvalid.jsx" be changed? SInce, the id='default' which is the same as in "DateTimePickerDefault.jsx".

File: packages/terra-date-time-picker/src/terra-dev-site/doc/guides/DateTimePickerEvents.jsx (hunk 2 above)
Comment (saket2403) [reply packages/terra-date-time-picker/src/terra-dev-site/doc/guides/DateTimePickerEvents.jsx]: Resolved in cc2c4e7

File: packages/terra-date-time-picker/src/terra-dev-site/doc/guides/DateTimePickerEvents.jsx (hunk 3 above)
Comment (saket2403) [reply packages/terra-date-time-picker/src/terra-dev-site/doc/guides/DateTimePickerEvents.jsx]: Resolved in cc2c4e7

**Comment by saket2403** [comment]
> Not sure, but should the IDs for DateTimePicker in the files "DateTimePickerIncomplete.jsx" and "DateTimePickerInvalid.jsx" be changed? SInce, the ID is 'default' which is the same as in "DateTimePickerDefault.jsx".

Changed in cc2c4e7

**Pull request merged**